platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/config/mapping/LdapLoginConfigMappingProvider.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.ldap.ldaplogin.config.mapping;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.codice.ddf.config.ConfigService;
+import org.codice.ddf.config.mapping.ConfigMapping;
+import org.codice.ddf.config.mapping.ConfigMapping.Id;
+import org.codice.ddf.config.mapping.ConfigMappingException;
+import org.codice.ddf.config.mapping.ConfigMappingProvider;
+import org.codice.ddf.config.model.LdapConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LdapLoginConfigMappingProvider implements ConfigMappingProvider {
+
+  private static final String LDAP_LOGIN_FACTORY_PID = "Ldap_Login_Config";
+  private static final String LDAPS = "LDAPS";
+  private static final String START_TLS_ENCRYPTION_CONFIG = "STARTTLS";
+
+  // configuration attribute names
+  private static final String LDAP_URL = "ldapUrl";
```

Comment:
Do some of those constants already exist in the current project? If so than we should reference them.